packages/terra-core-docs/src/terra-dev-site/test/pills/pill-list/CollapsiblePillListFirstEllipsesPill.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import { Button } from '@cerner/terra-docs';
+import PillList, { Pill } from 'terra-pills';
+import styles from './PillListTestCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const CollapsiblePillListFirstEllipsesPill = () => {
+  const [isCollapsed, setIsCollapsed] = useState(true);
+
+  const handleRollUpPillOnClick = () => {
+    setIsCollapsed(false);
+  };
+
+  const handleButtonOnClick = () => {
+    setIsCollapsed(!isCollapsed);
+  };
+
+  return (
+    <>
+      <PillList
+        ariaLabel="Pill Container"
+        isCollapsed={isCollapsed}
+        rollUpPillOnTrigger={handleRollUpPillOnClick}
```

Comment:
since `handleRollUpPillOnClick` will always sets it to false you can replace this function with `setIsCollapsed` like below 
```suggestion
        rollUpPillOnTrigger={setIsCollapsed}
```
this will assign !defaultValue of isCollapsed